x/genutil/client/cli/gentx.go

```diff
@@ -29,10 +33,18 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/staking/client/cli"
 )
 
+// ValidatorMsgBuildingHelpers helpers for message building gen-tx command
+type ValidatorMsgBuildingHelpers interface {
```

Comment:
Is there a better place this could live?

Reply:
yeah having interface definition in `cli` doesn't seem right to me, what about using `x/genutil/types`?

Reply:
That would introduce a dependency on pflag in `x/genutil/types` (see `*flag.FlagSet` below), which should remain IMHO CLI-library agnostic.

Reply:
oh right, I see.